api/schemas/PlaceData.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
+description: More informations about a place
+# TODO decide which geocoding API we use and have this resource match its type
+properties:
+  address:
+    type: string
+    example: via Garibaldi, 1
+  city:
+    type: string
+    example: Padova
+  zipcode:
+    type: string
+    minLength: 5
+    maxLength: 5
+  state:
+    type: string
+required:
+  - address
+  - city
+  - zipcode
+  - state
```

Comment:
In tutti gli altri schemi i required sono prima dell'elenco delle proprietà. Non cambia nulla, ma stona

Reply:
ti prego fammi sempre tu le review. stavo riordinando i required e questa mi era sfuggita, se fosse rimasta così avrei sofferto anch'io